Scripting/Bindings.cs

```diff
@@ -294,16 +574,16 @@ public override void _Input(InputEvent Event)
 					switch(Binding.AxisDirection)
 					{
 						case(DIRECTION.UP):
-							Scripting.ConsoleEngine.Execute($"{Binding.Function}({(float)new decimal (GreaterEqualZero(MotionEvent.Relative.y*-1))})", Scripting.ConsoleScope);
+							Scripting.ConsoleEngine.Execute($"{Binding.Function}({((float)new decimal (GreaterEqualZero(MotionEvent.Relative.y*-1)))/2})", Scripting.ConsoleScope);
```

Comment:
Can we make this `2` (for slowing mouse before applying look sensitivity so that joystick look is not painfully slow) a public static float in `Game` and increase it to equal the initial value of the look sensitivity (actually feed look sensitivity into this value as the default value so that it cannot be forgotten when changing the default value of the look sensitivity)